dash-pipeline/bmv2/stages/trusted_vni.p4

```diff
@@ -0,0 +1,48 @@
+#ifndef _DASH_STAGE_TRUSTED_VNI_P4_
+#define _DASH_STAGE_TRUSTED_VNI_P4_
+
+control trusted_vni_stage(
+    inout headers_t hdr,
+    inout metadata_t meta)
+{
+    action permit() {}
+
+    action deny() {
+        meta.dropped = true;
+    }
+
+    @SaiTable[single_match_priority = "true", api = "dash_trusted_vni", order=1, isobject="false"]
+    table global_trusted_vni {
+        key = {
+            meta.rx_encap.vni: range @SaiVal[name = "vni_range"];
+        }
+
+        actions = {
+            permit;
+        }
+    }
+
+    @SaiTable[single_match_priority = "true", api = "dash_trusted_vni"]
+    table trusted_vni {
+        key = {
+            meta.eni_id : exact @SaiVal[type="sai_object_id_t"];
+            meta.rx_encap.vni: range @SaiVal[name = "vni_range"];
+        }
+
+        actions = {
+            permit;
+            @defaultonly deny;
+        }
+        const default_action = deny;
+    }
+
+    apply {
+        if (global_trusted_vni.apply().hit) {
+            return;
+        }
+
+        trusted_vni.apply();
```

Comment:
need counter to track the drops.

Reply:
Add counter `eni_trusted_vni_entry_miss_drop`